### PR TITLE
Readme Edits

### PR DESCRIPTION
**Pull request by AsfahanX**
Content for non-telepathic human.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # hello-world
-Just another repository
+
+Hi Humans
```
Comment (asfahann) [inline README.md]: Good ideas here

Comment (asfahann) [inline README.md]: mantapp tuh

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # hello-world
-Just another repository
```
Comment (AsfahanX) [inline README.md:2]: Ngp di buang ?

Comment (AsfahanX) [inline README.md:2]: okk

**Review by AsfahanX — COMMENTED** [review]
sipp k

**Pull request merged**